### PR TITLE
[master < T946] Add update of the do module to docs

### PR DESCRIPTION
**Pull request by Josipmrden**
### Description

Update of the do module has been made since it has been rewritten in C++ for performance. Additional comments have been made in the module

### Pull request type

Please delete options that are not relevant and check the ones that are.

- [X] Documentation fix

### Checklist:

- [X] I checked all content with Grammarly
- [X] I performed a self-review of my code
- [X] I made corresponding changes to the rest of the documentation
- [X] The build passes locally
- [X] My changes generate no new warnings or errors

### CODE REVIEW

File: mage/query-modules/cpp/conditional-execution.md [hunk 1]
```diff
@@ -70,6 +70,12 @@ Parameters are prefixed with `$` like `$param_name`. For examples, see
 
 * `value: Map` ➡ Contains the result record of the executed query. Each `value` corresponds to one result record.
 
+:::caution
+Currently we only support returning records which contain `string`, `integer`, `double` or `boolean` primitives.
+Queries in the `conditionals` or in the `else_query` arguments which get executed and return other types of data, will
+result in an exception.
```
Comment (vpavicic) [inline mage/query-modules/cpp/conditional-execution.md:76]:
```suggestion
Currently, the module supports only those returning records that contain `string`, `integer`, `double` or `boolean` primitives.
Queries in the `conditionals` or in the `else_query` arguments, which get executed and return other types of data, will
result in an exception.
```

File: mage/query-modules/cpp/conditional-execution.md [hunk 2]
```diff
@@ -105,6 +111,12 @@ Parameters are prefixed with `$` like `$param_name`. For examples, see
...
```
Comment (vpavicic) [inline mage/query-modules/cpp/conditional-execution.md:117]:
```suggestion
Currently, the module supports only those returning records that contain `string`, `integer`, `double` or `boolean` primitives.
Queries in the `conditionals` or in the `else_query` arguments, which get executed and return other types of data, will
result in an exception.
```

**Comment by vpavicic** [comment]
@kgolubic can you please create a redirect from a python URL to the new, C++ one ... thnx! :)

**Comment by kgolubic** [comment]
@vpavicic I've checked, and we have that Python link only in the changelog. I've updated the link in the changelog to show to the new documentation page.

**Review by vpavicic — APPROVED** [review]

**Pull request merged**